From ff6a3353469246d67c0d93fc66d7fb5cba523918 Mon Sep 17 00:00:00 2001
From: Joe Talerico <jtaleric@redhat.com>
Date: Tue, 18 Oct 2016 12:01:27 -0400
Subject: [PATCH 1/3] Disable Neutron agents with OVN.

OVN natively implements services that are provided by Neutron agents.
This patch disables the Neutron DHCP agent as well as the OVS agent
for compute nodes.

Closes-bug: 1634580

Change-Id: I70631c2facbbf08257868e26e14af942ad7f2893
---
 environments/neutron-ml2-ovn.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/environments/neutron-ml2-ovn.yaml b/environments/neutron-ml2-ovn.yaml
index 821ad0c..bafb2a7 100644
--- a/environments/neutron-ml2-ovn.yaml
+++ b/environments/neutron-ml2-ovn.yaml
@@ -5,6 +5,9 @@ resource_registry:
   OS::TripleO::Services::NeutronOvsAgent: OS::Heat::None
   OS::TripleO::Services::NeutronCorePlugin: OS::TripleO::Services::NeutronCorePluginML2OVN
   OS::TripleO::Services::ComputeNeutronCorePlugin: ../puppet/services/neutron-compute-plugin-ovn.yaml
+# Disabling Neutron services that overlap with OVN
+  OS::TripleO::Services::NeutronDhcpAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronOvsAgent: OS::Heat::None
 
 parameter_defaults:
   NeutronMechanismDrivers: ovn
-- 
2.7.4

